test/gtest/ucp/test_ucp_ep_reconfig.cc

```diff
@@ -449,25 +457,43 @@ class test_reconfig_asymmetric : public test_ucp_ep_reconfig {
         sender().connect(&receiver(), get_ep_params());
         receiver().connect(&sender(), get_ep_params());
     }
+
+    ucp_tl_bitmap_t tl_bitmap() override
+    {
+        return ucp_tl_bitmap_max;
+    }
+
+    void init() override
+    {
+        static const std::vector<std::string> ib_tls = {"rc_mlx5", "dc_mlx5",
+                                                        "rc_verbs", "ud_verbs",
+                                                        "ud_mlx5"};
+
+        test_ucp_ep_reconfig::init();
+        bool has_ib = std::any_of(ib_tls.begin(), ib_tls.end(),
+                                  [&](const std::string &tl_name) {
+                                      return has_resource(sender(), tl_name);
+                                  });
+
+        if (!has_ib) {
+            /* In case there's no IB devices, new config will be identical to
+             * old config (thus no reconfiguration will be triggered). */
+            UCS_TEST_SKIP_R("No IB devices found");
+        }

```

Comment:
minor: so maybe assert this statement in the test?

Reply:
you mean instead of skipping it?